src/excmds.ts

```diff
@@ -3443,7 +3443,7 @@ export async function hint(option?: string, selectors?: string, ...rest: string[
 
         case "-W":
             selectHints = hinting.pipe(
-                DOM.HINTTAGS_selectors,
+                selectors || DOM.HINTTAGS_selectors,
                 elem => {
                     // /!\ RACY RACY RACY!
                     run_exstr(selectors + " " + rest.join(" ") + " " + elem)
```

Comment:
As I mentioned in Riot - we'll need to think of some way to tell if selectors are actually selectors or really part of rest[].